data/json/items/magazine/66mm.json

```diff
@@ -4,6 +4,7 @@
     "//": "Estimated from photographs and known length of launcher.",
     "looks_like": "belt40mm",
     "type": "MAGAZINE",
+    "category": "ammo",
```

Comment:
M74 doesn't appear to allow single-loading rockets, the clip holds them all and as such is a magazine.